tasks/build/index.js

```diff
@@ -15,15 +15,17 @@ module.exports = function (grunt) {
       '_build:packageJson',
       '_build:readme',
       '_build:babelCache',
+      '_build:packages',
       '_build:installDependencies',
+      'clean:packages',
       '_build:notice',
       '_build:removePkgJsonDeps',
       'clean:testsFromModules',
       'clean:examplesFromModules',
       '_build:copyNodeForOptimize',
       'run:optimizeBuild',
       'stop:optimizeBuild',
-      'clean:nodeForOptimize',
+      grunt.option('skip-clean') ? [] : ['clean:nodeForOptimize'],
```

Comment:
This is temporary and just simplifies testing this (as it doesn't remove the locally downloaded node binaries)